herddb-core/src/main/java/herddb/core/TableManager.java

```diff
@@ -184,8 +181,7 @@
     /**
      * Local locks
      */
-    private final ILocalLockManager locksManager =
-            USE_LEGACY_LOCK_MANAGER ? new LegacyLocalLockManager() : new LocalLockManager();
+    private ILocalLockManager locksManager = new LocalLockManager();
```

Comment:
please leave it final, it is better for the JVM for optimizations and in order to prevent NPEs